test-v13/build.gradle

```diff
@@ -0,0 +1,49 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
+
+android {
+    compileSdkVersion COMPILE_SDK_VERSION
+    buildToolsVersion BUILD_TOOLS_VERSION
+    defaultConfig {
+        applicationId "permissions.dispatcher.test_v13"
+        targetSdkVersion TARGET_SDK_VERSION
+        minSdkVersion 18
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_7
+        targetCompatibility JavaVersion.VERSION_1_7
+    }
+    lintOptions {
+        abortOnError false
+    }
+    sourceSets {
+        main.java.srcDirs += "src/main/kotlin"
+    }
+    packagingOptions {
+        exclude 'mockito-extensions/org.mockito.plugins.MockMaker'
```

Comment:
Yea, but without this setting, I can run `./gradlew check`, so let's delete this.